src/routes/leaderboard.tsx

```
@@ -23,11 +23,74 @@ import TrendingUp from "lucide-react/dist/esm/icons/trending-up";
+    club_id: string;
+    club_name?: string;
+    target_checkins: number;
+    current_checkins: number;
+    reward_points: number;
+    expires_at: string;
+  } | null>(null);
+
+  useEffect(() => {
+    const supabase = createClient();
+    (async () => {
+      const { data: { user } } = await supabase.auth.getUser();
```

ESLint

[error] Replace `·data:·{·user·}` with `⏎········data:·{·user·},⏎·····`

(prettier/prettier)